cclib/parser/logfileparser.py

```diff
@@ -124,6 +97,46 @@ def seek(self, pos: int, ref: int) -> None:
             self.pos = self.size
 
 
+def opencompressedfile(filename, mode = "r", object = None, wrap = False):
```

Comment:
Let's start trying to use type annotations, even if we don't check them and they aren't necessarily correct the first time, they can be refined later; I'm not sure about `object` or the return type:
```suggestion
def opencompressedfile(filename: str, mode: str = "r", object: typing.Optional[typing.IO] = None, wrap: bool = False) -> typing.IO:
```

Reply:
Oh, it was not so much about the argument being called `object` despite overwriting a built-in...it's just that I don't know what the type is here.

I personally annotate all functions, no matter their visibility.